app/utils/SummaryListRowHelper.scala

```diff
@@ -66,35 +66,49 @@ private[utils] class SummaryListRowHelper(implicit messages: Messages) {
       label = messages(s"$prefix.checkYourAnswersLabel", args: _*),
       answer = answer,
       id = id,
-      call = call,
+      call = Some(call),
       args = args: _*
     )
 
+  protected def buildRowWithNoChangeLink(
```

Comment:
Again is this used anywhere or for future use s I can only see if referred to in the AnswerHelper